backend/nomad_config.go

```diff
@@ -28,6 +28,8 @@ var (
 
 	flagNomadClientKey = flag.String("nomad-client-key", "", "Path to the Nomad Client Key File. "+
 		"Overrides the NOMAD_CLIENT_KEY environment variable if set. "+flagDefault(defaultConfig.NomadClientKey))
+
+	flagNomadHideEnvData = flag.String("hide-env-data", "", "Path to hideEnv bool. "+flagDefault(defaultConfig.HideEnvData))
```

Comment:
`Whether Nomad env{} values should be hidden (will prevent updating jobs in the UI)`